### ISSUE TITLE
Modifier keys no longer shared between keyboards in 10.9?

### ISSUE THREAD

**Issue by ryanbressler** [tekezo/Karabiner#274]
I discovered karabiner via one of several forum posts that recommend it as a way to share modifier keys between external keyboards for use as an ergonomic/split keyboard as documented in:

https://pqrs.org/osx/karabiner/document.html.en#implicit

However, at least on os x 10.9 with the latest download this behavior doesn't work for me. Has it been removed or moved to an option or something or is this a bug? Other keyboard remapping on a single keyboard does seem to work.

**Comment by gramirezjuan** [comment]
I have a similar problem, but both my keyboards are working. I'm running OSX 10.9.5 and want to share modifiers between my Apple wireless keyboard and Belkin yourtype keypad. The problem I was having was that Osx recognizes the keypad as another keyboard so it won't allow shortcuts to be applied if I press the modifiers in the apple keyboard and other keys in the keypad. Since an implicit behavior of Karabiner is to share modifier keys with all connected keyboards I thought installing it would solve the problem, but it hasn't worked. I verified that Karabiner was working by briefly enabling a change of key in the apple keyboard and that worked fine. I also verified that both keyboards appeared under "devices" in Event viewer. As of now I have nothing enabled in Karabiner.

Is there anything that you guys could recommend to make this work? Any help would be appreciated.
